protodroid-no-op/src/main/java/id/lukasdylan/grpc/protodroid/DataState.kt

```diff
@@ -0,0 +1,23 @@
+package id.lukasdylan.grpc.protodroid
+
+@Suppress("unused")
+data class DataState(
```

Comment:
So this class will be exposed to outside so app that used REST Networking will use this class for their logging right?? I think better to rename it with adding prefix "Protodroid" to make a unique class

Reply:
Sounds good!